projects/cdk/directives/value-changes/test/value-changes.directive.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {FormControl, FormGroup, ReactiveFormsModule} from '@angular/forms';
+import {By} from '@angular/platform-browser';
+import {TuiValueChangesModule} from '@taiga-ui/cdk';
+import {configureTestSuite} from '@taiga-ui/testing';
+
+describe(`TuiValueChangesDirective`, () => {
+    @Component({
+        template: `
+            <form
```

Comment:
<img width="255" alt="image" src="https://user-images.githubusercontent.com/12021443/230301717-e27b2e67-e0b8-4e24-be89-a3f9f140c8f4.png">

What do you think is it necessary to add an example here?